packages/config/src/layer2s/nil.ts

```diff
@@ -0,0 +1,24 @@
+import { upcoming } from './templates/upcoming'
+import { Layer2 } from './types'
+
+export const nil: Layer2 = upcoming({
+    id: '=nil;',
+    display: {
+        name: '=nil;',
+        slug: 'nil',
+        description:
+            "=nil; is a zkRollup that securely scales Ethereum to 60,000+ TPS through zkSharding, empowering web3 developers to build scalable, secure, and composable applications.",
```

Comment:
```suggestion
            "=nil; is a zkRollup that securely scales Ethereum through zkSharding, empowering web3 developers to build scalable and composable applications.",
```
please don't add claims that cannot be proven yet :p

Reply:
Sounds reasonable, thank you for the quick review!